### DISCUSSION TITLE
You said talk here.

### DISCUSSION THREAD

**Discussion by OldManKain** [HolographicWings/TOTK-Mods-collection#131] [General]
It would be nice if you would give the proper credit to theboy181 for all the M&M er Sweetmini exefs mods since they were stolen and credits are not given to the real creator,  I can provide proof as needed that the exefs mods were made way before M&M er sweetmini  posted them on a **Pirate** reddit (btw he can do romfs mods but doesn't know shit about offsets or ASM)   just do what is right.   It would be very appreciated.

**Comment by HolographicWings** [comment]
> It would be nice if you would give the proper credit to theboy181 for all the M&M er Sweetmini exefs mods since they were stolen and credits are not given to the real creator, I can provide proof as needed that the exefs mods were made way before M&M er sweetmini posted them on a **Pirate** reddit (btw he can do romfs mods but doesn't know shit about offsets or ASM) just do what is right. It would be very appreciated.

Firstly, you are harassing us here and on the discord and accuse me to removed something that i didn't
Secondly, i just noticed theses following thing ↓ so i'm not going to be nice with you

![image](https://github.com/HolographicWings/TOTK-Mods-collection/assets/19623115/6e4cad3c-5ab4-49cb-a3c7-aad6cb12e025)
![image](https://github.com/HolographicWings/TOTK-Mods-collection/assets/19623115/bda9250f-90d4-42f5-8501-b3ae7bc57ac7)

https://github.com/HolographicWings/TOTK-Mods-collection/blob/main/Mods/Graphics/Disable%20Targeting%20DOF/config.yaml
![image](https://github.com/HolographicWings/TOTK-Mods-collection/assets/19623115/2d60f8df-2049-42b0-ad86-09d21a5f31a3)
https://github.com/HolographicWings/TOTK-Mods-collection/blob/main/Mods/Graphics/Disable%20Targeting%20DOF/exefs/1.1.2.pchtxt
![image](https://github.com/HolographicWings/TOTK-Mods-collection/assets/19623115/a9b858e3-f002-45bc-84cd-9afd8d1b17ff)

https://github.com/HolographicWings/TOTK-Mods-collection/blob/main/Mods/Graphics/Island%20Fix/config.yaml
![image](https://github.com/HolographicWings/TOTK-Mods-collection/assets/19623115/550cd0b1-d891-4158-99b3-f0bcf37ed069)
https://github.com/HolographicWings/TOTK-Mods-collection/blob/main/Mods/Graphics/Island%20Fix/exefs/1.1.2.pchtxt
![image](https://github.com/HolographicWings/TOTK-Mods-collection/assets/19623115/aa34963b-80df-4ff7-865e-a985ccb0a536)

https://github.com/HolographicWings/TOTK-Mods-collection/blob/main/Mods/Graphics/LOD%20Improvement/config.yaml
![image](https://github.com/HolographicWings/TOTK-Mods-collection/assets/19623115/d0c8d6c9-d74b-47b9-90e2-5c43bf407e3c)
https://github.com/HolographicWings/TOTK-Mods-collection/blob/main/Mods/Graphics/LOD%20Improvement/exefs/1.1.2.pchtxt
![image](https://github.com/HolographicWings/TOTK-Mods-collection/assets/19623115/5e7e582d-8c3e-4baf-96e4-7d8eeb2fddb0)

https://github.com/HolographicWings/TOTK-Mods-collection/blob/main/Mods/Graphics/Remove%20Lens%20Flare/config.yaml
![image](https://github.com/HolographicWings/TOTK-Mods-collection/assets/19623115/833b24ec-bf61-4175-bd0d-15958d5137a6)
https://github.com/HolographicWings/TOTK-Mods-collection/blob/main/Mods/Graphics/Remove%20Lens%20Flare/exefs/1.1.2.pchtxt
![image](https://github.com/HolographicWings/TOTK-Mods-collection/assets/19623115/1219b7c8-0bd6-4b2c-94d7-a2b5ee3cb7fa)

Are you fucked up, blind, or just a troll ?
You harassed by pinging me on discord and now writing shit here
You didn't even read the credits
Well what is your answer ?
I'm not supposed to trashtalk on github but you are pissing me off

P.S. are you aware that theboy only do the mods for 1.1.0 ? can you tell me why there are 1.0.0, 1.1.1 and 1.1.2 versions too then ?
